ceph-dashboard-cephadm-e2e/config/definitions/ceph-dashboard-cephadm-e2e.yml

```diff
@@ -0,0 +1,84 @@
+- job:
+    name: ceph-dashboard-cephadm-e2e
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && focal && x86_64
+    display-name: 'ceph: Dashboard + Cephadm E2E'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
```

Comment:
As discussed with you about avoiding concurrent jobs in the same node, we'd need the [Throttle plugin](https://plugins.jenkins.io/throttle-concurrents/) installed and [configured in JJB](https://docs.openstack.org/infra/jenkins-job-builder/properties.html?highlight=throttle#properties.throttle).

Reply:
@djgalloway Can you help us here?

Reply:
`concurrent` means "at the same time anywhere."  The number of workers/executors per Jenkins builder determines how many jobs can run on a node at a time.

tl;dr - you want `concurrent: true`

Reply:
@djgalloway just the opposite: as per @alfonsomthd words, more than a single `kcli` instance running might cause errors. That Throttling plugin allows to set the number of concurrent jobs per node.

Reply:
@djgalloway As per our previous conversations we concluded that putting `concurrent: true` does not imply that more than 1 execution of the same job can run on the same machine, so it is safe to set it as true.